playbooks/ansible-test-cloud-integration-base/files/bootstrap-vcsa.yaml

```diff
@@ -0,0 +1,35 @@
+- hosts: appliance
+  gather_facts: false
+  vars:
+    ansible_password: zuul
```

Comment:
this isn't needed, we'll use SSH public keys to connection now

Reply:
I'm actually surprised it works. The password should have been changed.

Reply:
so, 3 things I can think of. 

1) we need to now disable ssh access using passwords, this is in sshd.conf
2) maybe we didn't save the config for passwords.
3) maybe it isn't even working, as ssh public keys get priority over password

Reply:
or SSH keeps using the same controlpath.